{{cookiecutter.project_slug}}/README.md

```diff
@@ -75,7 +79,7 @@ Django Admin that will be accesible at [http://localhost:8009/admin](http://loca
 
 To create a superuser use the following commands:
 
-    $ make shell
+    $ make shell-backend
```

Comment:
with a newly generated project it seems like `make shell-backend` no longer exists, and `make shell` doesn't work.

Reply:
I've reopened a ticket about that https://github.com/sixfeetup/scaf/issues/236